src/main/java/seedu/eylah/Eylah.java

```diff
@@ -178,27 +184,39 @@ public void run1() {
      * is not found,
      * or an personamount book will be used instead if errors occur when reading {@code storage}'s address book.
      */
-    private seedu.eylah.expensesplitter.model.Model initModelManager(PersonAmountStorage storage, ReadOnlyUserPrefs
+    private seedu.eylah.expensesplitter.model.Model initModelManager(Storage storage, ReadOnlyUserPrefs
         userPrefs) {
 
         Optional<ReadOnlyPersonAmountBook> personAmountBookOptional;
-        ReadOnlyPersonAmountBook initialData;
+        Optional<ReadOnlyReceiptBook> receiptBookOptional;
+        ReadOnlyPersonAmountBook initialPersonData;
+        ReadOnlyReceiptBook initialReceiptData;
         try {
 
             personAmountBookOptional = storage.readPersonAmountBook();
-            if (!personAmountBookOptional.isPresent()) {
+            receiptBookOptional = storage.readReceiptBook();
+            if (personAmountBookOptional.isEmpty()) {
                 logger.info("Data file not found. Will be starting with a sample PersonAmountBook");
             }
-            initialData = personAmountBookOptional.orElseGet(SamplePersonAmountDataUtil::getSamplePersonAmountBook);
+            if (receiptBookOptional.isEmpty()) {
+                logger.info("Data file not found. Will be starting with a sample ReceiptBook");
+            }
+            initialPersonData = personAmountBookOptional
+                    .orElseGet(SamplePersonAmountDataUtil::getSamplePersonAmountBook);
+            initialReceiptData = receiptBookOptional.orElseGet(SampleReceiptDataUtil::getSampleReceiptBook);
         } catch (DataConversionException e) {
-            logger.warning("Data file not in the correct format. Will be starting with an empty PersonAmountBook");
-            initialData = new PersonAmountBook();
+            logger.warning("Data file not in the correct format. Will be starting with an empty "
+                    + "PersonAmountBook and ReceiptBook");
+            initialPersonData = new PersonAmountBook();
+            initialReceiptData = new ReceiptBook();
         } catch (IOException e) {
-            logger.warning("Problem while reading from the file. Will be starting with an empty PersonAmountBook");
-            initialData = new PersonAmountBook();
+            logger.warning("Problem while reading from the file. Will be starting with an empty "
+                    + "PersonAmountBook and ReceiptBook");
+            initialPersonData = new PersonAmountBook();
+            initialReceiptData = new ReceiptBook();
         }
 
-        return new seedu.eylah.expensesplitter.model.ModelManager(new Receipt(), initialData, userPrefs);
+        return new seedu.eylah.expensesplitter.model.ModelManager(initialReceiptData, initialPersonData, userPrefs);
```

Comment:
alright so instead of a newReceipt() everytime we start eylah, it will take in the initial receipt data?

Reply:
This is for testing purpose for now. It will be removed after deployment.